MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofill.kt

```diff
@@ -20,10 +20,10 @@ public interface AddressAutofill {
      * @param options Request options.
      * @return Result of the search request, one of error or value.
      */
-    public suspend fun suggestions(
+    public suspend fun reverseGeocoding(
```

Comment:
AddressAutofil may not support reverseGeocoding

Reply:
Initially, I renamed this method for consistency but now reverted this change since it's not needed.